packages/mgt-components/src/graph/graph.files.ts

```diff
@@ -1087,3 +1103,35 @@ export async function fetchNextAndCacheForFilesPageIterator(filesPageIterator) {
     cache.putValue(key, { files: filesPageIterator.value, nextLink: filesPageIterator._nextLink });
   }
 }
+
+/**
+ * retrieves the specified document thumbnail
+ *
+ * @param {string} resource
+ * @param {string[]} scopes
+ * @returns {Promise<string>}
+ */
+export async function getDocumentThumbnail(graph: IGraph, resource: string, scopes: string[]): Promise<CacheThumbnail> {
```

Comment:
this is awesome - but I don't see where the photo is actually cached?

Reply:
Isn't this being done inside mgt-get: https://github.com/microsoftgraph/microsoft-graph-toolkit/blob/66a5bbb6591e6260e95dbc00c0d06bcbe8dcef38/packages/mgt-components/src/components/mgt-get/mgt-get.ts#L392-L394. I reused the implementation of https://github.com/microsoftgraph/microsoft-graph-toolkit/blob/66a5bbb6591e6260e95dbc00c0d06bcbe8dcef38/packages/mgt-components/src/graph/graph.photos.ts#L50 for this new function.

Reply:
Good point - let's keep it as is